src/app.js

```diff
@@ -1,88 +1,167 @@
+  div.style.width = '100%';
+  div.style.backgroundSize = '100%';
+  div.style.color = 'white';
+  content.appendChild(div);
+  div.innerHTML += '<div id="modal01" class="modal" onclick="this.style.display=\'none\'"> <img id="img01" class="mx-auto d-block w-50"> <p id="caption" class="text-center text-dark font-weight-bold bg-light"></p> </div>';
 }
-
-
-nav.appendChild(h1)
-nav.appendChild(ul)
-
-var div = document.createElement("div");
-div.id = "img_div"
+const div = document.createElement('div');
+div.id = 'img_div';
 div.classList.add('test');
 div.style.backgroundImage = "url('https://images.anovaculinary.com/sous-vide-salmon-2/directions/sous-vide-salmon-2-directions-image-1.jpg')";
-div.style.backgroundRepeat = "no-repeat";
-div.style.height = "100vh";
-div.style.width = "100%";
-div.style.backgroundSize = "100%";
-div.style.color = "white";
+div.style.height = '130vh';
+div.style.width = '100%';
+div.style.backgroundSize = '100%';
+div.style.color = 'white';
 content.appendChild(div);
 
-var textDiv = document.createElement("p")
-textDiv.id = "text_div"
-textDiv.appendChild(document.createTextNode("The best food in the world"));
-textDiv.style.fontSize = "80px";
-textDiv.style.textAlign = "center";
-textDiv.style.position = "absolute";
-textDiv.style.top = "50%";
-textDiv.style.left = "50%";
-textDiv.style.transform = "translate(-50%, -50%)";
-div.appendChild(textDiv)
-
-li1.onclick = function(){
-	document.getElementById("text_div").outerHTML = "";
-	var menu = document.createElement("div")
-	img_div = document.getElementById("img_div")
-	img_div.appendChild(menu)
-	var card1 = document.createElement("img")
-	card1.src = 'https://www.chimuadventures.com/blog/wp-content/uploads/2016/06/Ceviche_Peru_shutterstock1.jpg';
-	card1.style.height = "100px"
-	card1.style.width = "100px"
-	card1.style.objectFit = "cover";
-	menu.appendChild(card1)
-	var card2 = document.createElement("img")
-	card2.src = 'https://www.chimuadventures.com/blog/wp-content/uploads/2016/06/Ceviche_Peru_shutterstock1.jpg';
-	card2.style.height = "100px"
-	card2.style.width = "100px"
-	card2.style.objectFit = "cover";
-	menu.appendChild(card2)
-	var card3 = document.createElement("div")
-	var card4 = document.createElement("div")
-	var card5 = document.createElement("div")
-	var card6 = document.createElement("div")
+const textDiv = document.createElement('p');
+textDiv.id = 'text_div';
+textDiv.appendChild(document.createTextNode('an experience that you will never forget'));
+textDiv.style.fontSize = '80px';
+textDiv.style.textAlign = 'center';
+textDiv.style.position = 'absolute';
+textDiv.style.top = '50%';
+textDiv.style.left = '50%';
+textDiv.style.transform = 'translate(-50%, -50%)';
+div.appendChild(textDiv);
+
+
+div.innerHTML += '<div id="modal01" class="modal" onclick="this.style.display=\'none\'"> <img id="img01" class="mx-auto d-block w-50"> <p id="caption" class="text-center text-dark font-weight-bold bg-light"></p> </div>';
+
+function onClick(element) {
```

Comment:
- [ ] You should use **arrow functions** here instead of **traditional keywords** to be compliant with `ES6` standards.